src/components/input/InputRoot.tsx

```diff
@@ -27,9 +32,12 @@ const InputRoot = function InputRoot({
   classes,
   elementRef,
   hasError,
+  feedback,
 
   ...htmlAttributes
 }: InputRootProps) {
+  const inputFeedback = feedback ?? (hasError ? 'error' : undefined);
```

Comment:
You could also just re-assign `feedback` here since arguments are mutable.

Reply:
Hah, I didn't think about that. I'm too used to not reassigning stuff 😅